### ISSUE TITLE
feat: add new modules written in rust and combine with blockchain

### ISSUE THREAD

**Issue by cymqqqq** [apitable/apitable#1203]
I have an idea for this project.
I have read some backend and frontend code here, which are written in Java and typescript.
So my proposal is: Does the apitable have any plans to add new modules(micro-services) written in Rust? Or how to combine it with blockchain.
The action code I submit includes how to deploy the compiled frontend code in the ICP network(It's like decentralized AWS, a blockchain protocol, developed by the Dfinity Foundation).

## Why? 
> Now, the micro-services written in Rust have many advantages, more efficient. Rust, a compiled language, have inherent performance benefits attached to them. In serverless applications, this usually means faster runtime and lower memory usage, which translates into better service and lower costs.
> What is the user story for this feature? 
Here is the story I have ever found(Personally). (https://medium.com/tenable-techblog/building-a-microservice-with-rust-23a4de6e5e14)

## What?
> Can you describe this feature in detail?
1. the detailed backend task is written in Rust.
2. the backend communicates with the database and cache(Read and Write).
3. To build web API.(frontend can call API interface from backend).
4. To dockerize the backend micro-services.
> Who can benefit from it? 
Developers and Consumers

**Comment by zacker330** [comment]
It's a huge decision to move from Typescript and Java to Rust, we have to trade off. Can you list the pros and cons one by one?

**Comment by cymqqqq** [comment]
> It's a huge decision to move from Typescript and Java to Rust, we have to trade off. Can you list the pros and cons one by one?

Yes, it's a huge decision, now I'm reading the source code, and maybe I can list the pros and cons later.

**Comment by mr-kelly** [comment]
Thank you for the proposal and for sharing your thoughts on integrating Rust and blockchain technologies like ICP! While we truly appreciate the innovative ideas and community brainstorming, we currently do not have any plans to rewrite our core microservices in Rust or integrate with blockchain networks. Therefore, we will close this issue for now, but we always welcome your feedback and continued engagement in our community.
